### ISSUE TITLE
Linux: Fragpipe Headless Thermo Native Libraries Not Detected

### ISSUE THREAD

**Issue by Sanpme66** [Nesvilab/FragPipe#1605]
** Log file**

[Fragpip_Linux.zip](https://github.com/user-attachments/files/15528494/Fragpip_Linux.zip)

**Issue Description:**
Thank you for the FragPipe Linux CLI release.

I'm encountering an error related to raw file I/O; it appears that the "Batmass-IO binaries for Thermo support and/or Thermo native libraries" are not found.

Steps taken:
- Downloaded **MSFragger-4.0** and included its directory path using `--config-msfragger`.
- Did the same for `--config-philosopher` and `--config-python`, following a similar resolution from someone who encountered a similar error. **MSFragger-4.0** contains Thermo and Bruker libraries.

The raw file is from Thermo Fisher Mass Spectra.

Command used in the shell:
```shell
./bin/fragpipe --headless --workflow /canelib/raw_data/tools/fragpipe/test/data/output/fragpipe.workflow --manifest /canelib/raw_data/tools/fragpipe/test/data/output/fragpipe-files.fp-manifest --workdir /canelib/raw_data/tools/fragpipe/test/data/output --config-philosopher /canelib/raw_data/tools/fragpipe/tools/philosopher --config-python /canelib/raw_data/tools/fragpipe/py3.9/bin/python3 --config-msfragger /canelib/raw_data/tools/fragpipe/tools/MSFragger-4.0/MSFragger-4.0.jar > test/data/output/logs.log
```

Could you please review the log file attached?

Thank you!

**Comment by fcyu** [comment]
Have you installed mono according to the tutorial here https://fragpipe.nesvilab.org/docs/tutorial_setup_fragpipe.html ?

If yes, could you run `mono /canelib/raw_data/tools/fragpipe/tools/MSFragger-4.0/ext/thermo/BatmassIoThermoServer.exe`, and send me the screenshot?

Thanks,

Fengchao

**Comment by Sanpme66** [comment]
Yes! You're correct. The `mono` toolkit was missing during installation, which caused the issue. It's now up and running smoothly. 

Thanks for the response!



> Have you installed mono according to the tutorial here https://fragpipe.nesvilab.org/docs/tutorial_setup_fragpipe.html ?
> 
> If yes, could you run `mono /canelib/raw_data/tools/fragpipe/tools/MSFragger-4.0/ext/thermo/BatmassIoThermoServer.exe`, and send me the screenshot?
> 
> Thanks,
> 
> Fengchao
